tests/test_benchmark_simple_scenarios.py

```diff
@@ -261,6 +258,50 @@ def test_benchmark_AE_grid_battery_peak_pricing(self, margs):
                 ):
                     assert abs(battery_charge[i - 1]) > 0
 
+    @pytest.mark.skipif(
+        EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
+        reason="Benchmark test deactivated, set env variable "
+        "EXECUTE_TESTS_ON to 'master' to run this test",
+    )
+    @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
+    def test_benchmark_AFG_grid_heatpump_heat(self, margs):
+        use_case = "AFG_grid_heatpump_heat"
+        main(
+            overwrite=True,
+            display_output="warning",
+            path_input_folder=os.path.join(TEST_INPUT_PATH, use_case),
+            input_type=CSV_EXT,
+            path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
+        )
+        # read json with results file
+        with open(
+            os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json"), "r"
+        ) as results:
+            data = json.load(results)
+        # read excel sheet with time series
+        busses_flow = pd.read_excel(
+            os.path.join(TEST_OUTPUT_PATH, use_case, "timeseries_all_busses.xlsx"),
+            sheet_name=bus_suffix("Heat"),
+        )
+        # create dict with electricity prices
+        electricity_price = data[ENERGY_PROVIDERS]["Electricity DSO"][ENERGY_PRICE][
+            VALUE
+        ]["data"]
+        # compare cost of using heat pump with electricity price to heat price
+        for i in range(0, len(electricity_price)):
+            if (
+                electricity_price[i]
+                / data[ENERGY_CONVERSION]["heat_pump"][EFFICIENCY][VALUE]
+                > data[ENERGY_PROVIDERS]["Heat DSO"][ENERGY_PRICE][VALUE]
+            ):
+                assert busses_flow["Gas_consumption_period"][i] == approx(
+                    abs(busses_flow["demand_heat"][i])
+                )
```

Comment:
We can make the assertions easier to understand by adding an assertion error message. This will help to fix the benchmark tests later.

```suggestion
                assert busses_flow["Gas_consumption_period"][i] == approx(
                    abs(busses_flow["demand_heat"][i])
                ), f"Eventhough the marginal costs to use the heat pump are higher then the heat DSO price with {electricity_price[i]
                / data[ENERGY_CONVERSION]["heat_pump"][EFFICIENCY][VALUE]} comp. data[ENERGY_PROVIDERS]["Heat DSO"][ENERGY_PRICE][VALUE]}, the heat DSO is not solely used for energy supply."
                
I am a bit confused with the names here - is it a heat dso or a gas dso ("Gas_consumption_period") that we are talking about here? 
```